src/Symfony/Component/JsonPath/README.md

```diff
@@ -0,0 +1,13 @@
+JsonPath Component
+==================
+
+The JsonPath component eases JSON navigation by manipulating JSONPath as described in RFC 9535.
```

Comment:
```suggestion
The JsonPath component eases JSON navigation by manipulating JSON paths as described in RFC 9535.
```

Like this? And maybe also link to the RFC?

Reply:
I added the link to the RFC, good idea.

The RFC describes the "JSONPath syntax", so I tried an alternate version: `The JsonPath component eases JSON navigation using the JSONPath syntax`. What do you think?

Reply:
👍 this reads better